cos.c

```diff
@@ -0,0 +1,16 @@
+float coseno(float z){
```

Comment:
Funciona bien, solo tiene 2 warnings ;)

coseno.c:7:3: warning: incompatible implicit declaration of built-in function ‘printf’ [enabled by default]
   printf("\nIngrese el ángulo en radianes: ");
   ^
coseno.c:8:3: warning: incompatible implicit declaration of built-in function ‘scanf’ [enabled by default]
   scanf("%f", &z);
   ^